src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -1,15 +1,14 @@
 package seedu.duke.ui;
 
-import seedu.duke.CommandList;
-import seedu.duke.Formatter;
-import seedu.duke.Parser;
-import seedu.duke.SyntaxAnalyser;
+import seedu.duke.*;
```

Comment:
As mentioned by @runxinghuan , you shouldn't use *.

Reply:
Yes, I agree